### PR TITLE
Update Requirements

### LINKED ISSUE #1897: Upgrade PyXForm dependency
### Request

PyXForm has recently tagged a new release `v1.2.0.`(Latest is 1.3.*) we should upgrade to this version as soon as possible as it fixes some open issues #1817

### PR DESCRIPTION
**Pull request by DavisRayM**
### Changes / Features implemented

_Major Upgrades_

```
Celery 4.4.0 -> 5.0.0
Django RestFramework XML 1.4.0 -> 2.0.0
sphinx 2.4.2 -> 3.2.1
django-ses 0.8.9 -> 1.0.3
```
_Removed Packages_

```
django-celery-results
```

### CODE REVIEW

File: onadata/apps/main/models/audit.py [hunk 1]
```diff
@@ -74,9 +74,10 @@ def query_data(cls, username, query=None, fields=None, sort=None, start=0,
             or_params = []
             if '$or' in list(query):
                 or_dict = query.pop('$or')
-                for l in or_dict:
-                    or_where.extend([u"json->>%s = %s" for i in l.items()])
-                    [or_params.extend(i) for i in l.items()]
+                for or_query in or_dict:
+                    or_where.extend(
+                        [u"json->>%s = %s" for i in or_query.items()])
+                    [or_params.extend(i) for i in or_query.items()]
```
Comment (pld) [inline onadata/apps/main/models/audit.py:80]: Codacy found an issue: [Expression "[or_params.extend(i) for i in or_query.items()]" is assigned to nothing](https://app.codacy.com/gh/onaio/onadata/pullRequest?prid=6237366)

**Comment by pld** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/onaio/onadata/pullRequest?prid=6237366&bid=20321852)

**Comment by pld** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/onaio/onadata/pullRequest?prid=6237366&bid=20321852)

File: onadata/apps/main/models/audit.py (hunk 1 above)
Comment (pld) [inline onadata/apps/main/models/audit.py:80]: Codacy found an issue: [Expression "[or_params.extend(i) for i in or_query.items()]" is assigned to nothing](https://app.codacy.com/gh/onaio/onadata/pullRequest?prid=6237366)

File: onadata/libs/utils/csv_import.py [hunk 2]
```diff
@@ -489,21 +489,17 @@ def get_async_csv_submission_status(job_uuid):
...
-            response = async_status(celery_state_to_status(job.state))
-            if isinstance(job.info, dict):
-                response.update(job.info)
-
-            return response
+    # result = (job.result or job.state)
+    if job.state not in ['SUCCESS', 'FAILURE']:
+        response = async_status(celery_state_to_status(job.state))
+        if isinstance(job.info, dict):
+            response.update(job.info)
```
Comment (ivermac) [inline onadata/libs/utils/csv_import.py:496]: By removing the `try/except` block, does that mean we no longer need to catch `BacklogLimitExceeded`?

**Review by ivermac — COMMENTED** [review]
Single comment

File: onadata/apps/main/models/audit.py (hunk 1 above)
Comment (pld) [reply onadata/apps/main/models/audit.py:80]: you can add a inline comment to instruct codacy to ignore this

**Comment by pld** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/onaio/onadata/pullRequest?prid=6237366&bid=20321852)

File: onadata/apps/main/models/audit.py [hunk 3]
```diff
@@ -74,9 +74,12 @@ def query_data(cls, username, query=None, fields=None, sort=None, start=0,
             or_params = []
             if '$or' in list(query):
                 or_dict = query.pop('$or')
-                for l in or_dict:
-                    or_where.extend([u"json->>%s = %s" for i in l.items()])
-                    [or_params.extend(i) for i in l.items()]
+                for or_query in or_dict:
+                    or_where.extend(
+                        [u"json->>%s = %s" for i in or_query.items()])
+                    [
```
Comment (pld) [inline onadata/apps/main/models/audit.py:80]: Codacy found an issue: [Expression "[or_params.extend(i) for i in or_query.items()]" is assigned to nothing](https://app.codacy.com/gh/onaio/onadata/pullRequest?prid=6237366)

File: onadata/libs/utils/csv_import.py (hunk 2 above)
Comment (DavisRayM) [reply onadata/libs/utils/csv_import.py:496]: The `BacklogLimitExceeded` error that was handled here was raised by the `amqp` backend which is no longer present but I've seen the error is present in the `rpc` backend that's used by AMQP brokers. I think I'll re-add this for now and import the error from that backend instead

**Comment by pld** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/onaio/onadata/pullRequest?prid=6237366&bid=20321852)

**Comment by pld** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/onaio/onadata/pullRequest?prid=6237366&bid=20321852)

**Comment by pld** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/onaio/onadata/pullRequest?prid=6237366&bid=20321852)

**Comment by pld** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/onaio/onadata/pullRequest?prid=6237366&bid=20321852)

**Comment by pld** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/onaio/onadata/pullRequest?prid=6237366&bid=20321852)

**Review by ivermac — APPROVED** [review]

**Comment by chlarsen** [comment]
Celery 5 demands a new init script, because its syntax has changed.

**Pull request merged**